tools/program.py

```diff
@@ -33,6 +33,7 @@
 from ppocr.utils.save_load import save_model
 import numpy as np
 from ppocr.utils.character import cal_predicts_accuracy, cal_predicts_accuracy_srn, CharacterOps
+import paddleslim as slim
```

Comment:
Put `import` in line 303 so that there is no need to install paddleslim if not using model compression